Formula/libnghttp2.rb

```diff
@@ -1,10 +1,9 @@
 class Libnghttp2 < Formula
   desc "HTTP/2 C Library"
   homepage "https://nghttp2.org/"
-  url "https://github.com/nghttp2/nghttp2/releases/download/v1.53.0/nghttp2-1.53.0.tar.gz"
-  mirror "http://fresh-center.net/linux/www/nghttp2-1.53.0.tar.gz"
-  mirror "http://fresh-center.net/linux/www/legacy/nghttp2-1.53.0.tar.gz"
```

Comment:
removing this (feel this is a bit redundant)

Reply:
It's not redundant - note how `http://fresh-center.net/linux/www/nghttp2-1.53.0.tar.gz` no longer exists but `http://fresh-center.net/linux/www/legacy/nghttp2-1.53.0.tar.gz` does.

Reply:
Well right now before this PR is merged is a perfect example of when installing `libnghttp2` would be broken if legacy wasn't there.

Reply:
yeah, that is true 😆